css/index.styl

```diff
@@ -79,7 +79,7 @@ breakpoint(point, value = 0)
       transform none
 
 .auth0-lock.auth0-lock
-  font-family "Avenir", sans-serif
+  font-family "Avenir Next", Avenir, -apple-system, BlinkMacSystemFont, Roboto, Hevetica, sans-serif
```

Comment:
I assume that the Font Family declarations for Roboto and the other not-standard fonts is already in place.